### PR TITLE
[Hounslow] Add cobrand

### PR DESCRIPTION
**Pull request by davea**
This is the Hounslow cobrand including the various template & functionality tweaks made along the way.

This PR consists of all Hounslow-related work except for the general enquiries form.

### CODE REVIEW

File: perllib/FixMyStreet/Cobrand/Hounslow.pm [hunk 1]
```diff
@@ -80,4 +94,13 @@ sub should_skip_sending_update {
 }
 
 
+sub lookup_site_code_config { {
+    buffer => 50, # metres
+    url => "https://tilma.staging.mysociety.org/mapserver/hounslow",
```
Comment (struan) [inline perllib/FixMyStreet/Cobrand/Hounslow.pm]: This will need to be updated before going live.

File: web/cobrands/hounslow/assets.js [hunk 2]
```diff
@@ -0,0 +1,235 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+var defaults = {
+    http_options: {
+        url: "https://tilma.staging.mysociety.org/mapserver/hounslow",
```
Comment (struan) [inline web/cobrands/hounslow/assets.js]: likewise, these will need to be updated.

File: perllib/FixMyStreet/Cobrand/Hounslow.pm [hunk 3]
```diff
@@ -6,7 +6,7 @@ use warnings;
 
 sub council_area_id { 2483 }
 sub council_area { 'Hounslow' }
-sub council_name { 'Hounslow Borough Council' }
+sub council_name { 'Hounslow Highways' }
```
Comment (struan) [inline perllib/FixMyStreet/Cobrand/Hounslow.pm]: I wonder if at some point we want a display name and a body name but probably not worth it unless we get more of these.

File: web/cobrands/hounslow/js.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+var org_id = '5540';
+var body = "Hounslow Borough Council";
+fixmystreet.assets.add($.extend(true, {}, fixmystreet.roadworks.layer_future, {
+    http_options: { params: { organisation_id: org_id } },
```
[the remark this answers is not in the document]
Comment (davea) [reply web/cobrands/hounslow/js.js]: I think this was caused by a special character in the JSON response which was causing a parsing error:

```
Bad JSON: SyntaxError: Unexpected token  in JSON at position 166184
```

I've improved the JSON handling in 76071ab74923a064aac1aa8c1e3f7b34ffaa6f2d.

**Pull request merged**